### PR TITLE
Application list

### PR DESCRIPTION
**Pull request by hone1er**
### What changes were made?

- added proposal list and proposal cards to display on client project page

### Is there any background info about the change?

- Displays list of users that have applied for the job
- Proposals currently don't save the name of the user, which is needed in the design

### Are there any state changes? Mention key ones (if any)

- isClientPage tracks if page was opened by a client. Still need proper permissions so only the owner can see the applications, not just any client

### Screenshots or Gifs✨
Screen Shot 2022-08-04 at 8 23 18 AM

### Shortcut Link

https://app.shortcut.com/twali/story/1152/display-a-list-of-applicants-in-the-project-overview

### CODE REVIEW

File: components/Project/Project.tsx [hunk 1]
```diff
@@ -172,8 +188,85 @@ const Project = ({
                 <Flex flexDirection="column" width="100%">
                   <ProjectDescription />
                   {isProjectPage ? (
-                    // proposal box for expert to apply
-                    <ProjectProposalBox userData={userData} bounty={bountyState} setProposalStatus={setProposalStatus}/>
+                    <>
```
Comment (nagmak) [inline components/Project/Project.tsx:191]: Hey @hone1er looks like the Proposal Box Component was removed here - should have it back in!

**Review by nagmak — CHANGES_REQUESTED** [review]
Please add back in the ProjectProposalBox component! Once that's done this is good to merge :) Everything else looks great!

**Comment by hone1er** [comment]
> Please add back in the ProjectProposalBox component! Once that's done this is good to merge :) Everything else looks great!

Done 🫡✅ @nagmak

**Review by nagmak — APPROVED** [review]
Looks great!! 🚢

**Review by NickGonzalez04 — APPROVED** [review]
👍 to 🚢

**Pull request merged**